src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/ApprenticeshipUpdatesPartial/_ViewOriginalApprenticeshipForApprenticeshipUpdates.cshtml

```diff
@@ -11,6 +13,13 @@
                 <dt class="govuk-summary-list__key"> Unique learner number </dt>
                 <dd class="govuk-summary-list__value"> @Model.ULN</dd>
             </div>
+            @if (Model.DeliveryModel == DeliveryModel.Flexible)
```

Comment:
It would be better to test for "not equals normal"  as there will be other abnormal delivery models in the future.